plenum/server/replica.py

```diff
@@ -2337,21 +2353,21 @@ def dequeue_commits(self, viewNo: int, ppSeqNo: int):
         key = (viewNo, ppSeqNo)
         if key in self.commitsWaitingForPrepare:
             if not self.has_prepared(key):
-                self.logger.info('{} has not prepared {}, will dequeue the '
-                                 'COMMITs later'.format(self, key))
+                # self.logger.info('{} has not prepared {}, will dequeue the '
+                #                  'COMMITs later'.format(self, key))
                 return
             i = 0
             # Keys of pending prepares that will be processed below
             while self.commitsWaitingForPrepare[key]:
                 commit, sender = self.commitsWaitingForPrepare[
                     key].popleft()
-                self.logger.info("{} popping stashed COMMIT{}".format(self, key))
+                # self.logger.info("{} popping stashed COMMIT{}".format(self, key))
                 self.process_three_phase_msg(commit, sender)
 
                 i += 1
             self.commitsWaitingForPrepare.pop(key)
-            self.logger.info("{} processed {} COMMITs waiting for PREPARE for"
-                             " view no {} and seq no {}".format(self, i, viewNo, ppSeqNo))
+            # self.logger.info("{} processed {} COMMITs waiting for PREPARE for"
```

Comment:
Please leave it on INFO level (do not comment out)

Reply:
ok